docs/advanced.md

```diff
@@ -36,6 +36,42 @@ Alternatively, you can explicitly close the connection pool without block-usage
 
 Once you have a `Client`, you can use all the features documented in the [Quickstart](/quickstart) guide.
 
+### Configuration
+
+Clients allow you to apply configuration to all outgoing requests by passing parameters to the `Client` constructor.
+
+For example, to apply a set of custom headers on every request:
+
+```python
+>>> url = 'http://httpbin.org/headers'
+>>> headers = {'user-agent': 'my-app/0.0.1'}
+>>> with httpx.Client(headers=headers) as client:
+...     r = client.get(url)
+...
+>>> r.json()['headers']['User-Agent']
+'app-1.0'
```

Comment:
Just tried this snippet locally and the result is 'my-app/0.0.1'?

Reply:
Oh, right!